web/packages/plugins/analysis-platform/src/modules/Analysis/components/AnalysisInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react'
+import { Button, Tooltip } from '@equinor/eds-core-react'
+import Icons from '../../../components/Design/Icons'
+
+import { AuthContext, hasExpertRole } from '@dmt/common'
+import { TAnalysisInfoCardProps } from '../../../Types'
+import { CreateJobButton } from './CreateJobButton'
+import { DocumentInfoCard, OnRight } from '../../../components'
+
+const AnalysisInfoCardActions = (props: TAnalysisInfoCardProps) => {
+  const { analysis, addJob, jobs, dataSourceId } = props
+  const { tokenData } = useContext(AuthContext)
+  return (
+    <>
+      {'task' in analysis && Object.keys(analysis.task).length > 0 && (
```

Comment:
not a big deal, but it seems like hasOwnProperty('key') is recommended to check if an object has a given key

[(https://stackoverflow.com/questions/455338/how-do-i-check-if-an-object-has-a-key-in-javascript)](https://stackoverflow.com/a/455366)